include/xtensor/xcsv.hpp

```diff
@@ -31,6 +33,9 @@ namespace xt
     template <class T, class A = std::allocator<T>>
     xcsv_tensor<T, A> load_csv(std::istream& stream);
 
+    template <class T, class A = std::allocator<T>>
+    xcsv_tensor<T, A> read_csv(std::istream& stream);
```

Comment:
I think we should keep the same method name and dispatch according to a TAG or enum, something like:
```cpp
enum class csv_mode
{
    fast,
    strict
};
template <class T, class A = std::allocator<T>>
xcsv_tensor<T, A> load_csv(std::istream& stream, csv_mode mode = csv_mode::fast)
{
    // Calls the implementation method accoriding to the value of csv_mode
}
```